ghost/core/test/e2e-api/admin/pages.test.js

```diff
@@ -430,4 +430,33 @@ describe('Pages API', function () {
         res.body.should.be.empty();
         res.headers['x-cache-invalidate'].should.eql('/*');
     });
+
+    it('Can copy a page', async function () {
```

Comment:
We should not be making any more of these test using the old way of e2e testing - without snapshots. Lets take similar strategy here that we took with old posts tests - [posts-legacy.test.js](https://github.com/TryGhost/Ghost/blob/72ed8f56f641fc1ca81e4e363fcd282c195f95c0/ghost/core/test/e2e-api/admin/posts-legacy.test.js#L22). Lets rename this test suite to `pages-legacy.test` and than the new test in `pages.test.js` should be written with a snapshot.